### PR TITLE
QPPSF-4880 Fixing CAHPS Measures

### PR DESCRIPTION
**Pull request by Jpec07**
#### Motivation for change

CAHPS measure data for 2019 contained erroneous entries that were causing problems when developing for Scoring.

#### What is being changed

The updated CSV was retrieved from policy and imported. The resulting JSONs have been updated.

#### Release checklist:
Tasks that must be done prior to merging this PR, including testing.

* [x] [Package version updated](https://github.com/CMSgov/qpp-measures-data/blob/master/CONTRIBUTING.md#versioning-publishing-and-creating-new-releases)
* [x] Documentation updated
* [x] Unit tests added/passing
* [x] Verified working locally

##### Associated JIRA tickets:
* https://jira.cms.gov/browse/QPPSF-4880

### CODE REVIEW

File: measures/2019/measures-data.json [hunk 1]
```diff
@@ -10869,7 +10869,7 @@
...
+    "description": "Adopt a formal model for quality improvement and create a culture in which all staff actively participates in improvement activities that could include one or more of the following such as:  \r-  Multi-Source Feedback; \r-  Train all staff in quality improvement methods; \r-  Integrate practice change/quality improvement into staff duties; \r-  Engage all staff in identifying and testing practices changes; \r-  Designate regular team meetings to review data and plan improvement cycles; \r-  Promote transparency and accelerate improvement by sharing practice level and panel level quality of care, patient experience and utilization data with staff; and/or \r-  Promote transparency and engage patients and families by sharing practice level quality of care, patient experience and utilization data with patients and families, including activities in which clinicians act upon patient experience data.",
```
Comment (shadford) [inline measures/2019/measures-data.json]: I saw these "\r-" the last time I changed PI measures, thought is was a weird, it must be coming in from the generator code...

**Comment by Jeffreyher** [comment]
@Jpec07 Hey I just published a new version. You may need to bump your version again. srry.

File: measures/2019/measures-data.json (hunk 1 above)
Comment (Jpec07) [reply measures/2019/measures-data.json]: It's a Windows linebreak vs. a Unix linebreak (something that has been the bane of my existence as a dev on Windows).

File: staging/2019/measures-data-ia.json [hunk 2]
```diff
@@ -156,7 +156,7 @@
...
+    "description": "Empanel (assign responsibility for) the total population, linking each patient to a MIPS eligible clinician or group or care team.\rEmpanelment is a series of processes that assign each active patient to a MIPS eligible clinician or group and/or care team, confirm assignment with patients and clinicians, and use the resultant patient panels as a foundation for individual patient and population health management. \rEmpanelment identifies the patients and population for whom the MIPS eligible clinician or group and/or care team is responsible and is the foundation for the relationship continuity between patient and MIPS eligible clinician or group /care team that is at the heart of comprehensive primary care. Effective empanelment requires identification of the \"active population\" of the practice: those patients who identify and use your practice as a source for primary care. There are many ways to define \"active patients\" operationally, but generally, the definition of \"active patients\" includes patients who have sought care within the last 24 to 36 months, allowing inclusion of younger patients who have minimal acute or preventive health care.",
```
Comment (KyleApfel) [inline staging/2019/measures-data-ia.json]: You'll need to resolve these `\r` glitches before we can merge this into the measures data repo. It makes tracking issues in PR more difficult and could cause issues when displaying on the UI.

**Pull request merged**